worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala

```diff
@@ -95,54 +102,55 @@ class FetchHandler(
+  private def handleRpcRequest(
+      client: TransportClient,
+      rpcRequest: RpcRequest,
+      callback: RpcResponseCallback): Unit = {
+    var message: GeneratedMessageV3 = null
     try {
-      var message: GeneratedMessageV3 = null
-      try {
-        message = TransportMessage.fromByteBuffer(rpcRequest.body().nioByteBuffer())
-          .getParsedPayload[GeneratedMessageV3]
-      } catch {
-        case exception: CelebornIOException =>
-          logWarning("Handle request with legacy RPCs", exception)
-          return handleLegacyRpcMessage(client, rpcRequest)
-      }
-      message match {
-        case openStream: PbOpenStream =>
-          handleOpenStreamInternal(
-            client,
-            openStream.getShuffleKey,
-            openStream.getFileName,
-            openStream.getStartIndex,
-            openStream.getEndIndex,
-            openStream.getInitialCredit,
-            rpcRequest.requestId,
-            isLegacy = false,
-            openStream.getReadLocalShuffle)
-        case bufferStreamEnd: PbBufferStreamEnd =>
-          handleEndStreamFromClient(bufferStreamEnd.getStreamId, bufferStreamEnd.getStreamType)
-        case readAddCredit: PbReadAddCredit =>
-          handleReadAddCredit(readAddCredit.getCredit, readAddCredit.getStreamId)
-        case chunkFetchRequest: PbChunkFetchRequest =>
-          handleChunkFetchRequest(
-            client,
-            StreamChunkSlice.fromProto(chunkFetchRequest.getStreamChunkSlice),
-            rpcRequest)
-        case message: GeneratedMessageV3 =>
-          logError(s"Unknown message $message")
-      }
-    } finally {
-      rpcRequest.body().release()
+      message = TransportMessage.fromByteBuffer(rpcRequest.body().nioByteBuffer())
+        .getParsedPayload[GeneratedMessageV3]
+    } catch {
+      case exception: CelebornIOException =>
+        logWarning("Handle request with legacy RPCs", exception)
+        return handleLegacyRpcMessage(client, rpcRequest, callback)
+    }
+    message match {
+      case openStream: PbOpenStream =>
+        handleOpenStreamInternal(
+          client,
+          openStream.getShuffleKey,
+          openStream.getFileName,
+          openStream.getStartIndex,
+          openStream.getEndIndex,
+          openStream.getInitialCredit,
+          rpcRequest.requestId,
+          isLegacy = false,
+          openStream.getReadLocalShuffle,
+          callback)
+      case bufferStreamEnd: PbBufferStreamEnd =>
+        handleEndStreamFromClient(bufferStreamEnd.getStreamId, bufferStreamEnd.getStreamType)
+      case readAddCredit: PbReadAddCredit =>
+        handleReadAddCredit(readAddCredit.getCredit, readAddCredit.getStreamId)
+      case chunkFetchRequest: PbChunkFetchRequest =>
+        handleChunkFetchRequest(
+          client,
+          StreamChunkSlice.fromProto(chunkFetchRequest.getStreamChunkSlice),
+          rpcRequest)
+      case message: GeneratedMessageV3 =>
```

Comment:
is it better to make this case a general case?

```
case _ =>
```

Reply:
This is unrelated to this change so I don't want to change it.